.github/workflows/test.yml

```diff
@@ -35,3 +35,12 @@ jobs:
         run: |
           cd tests && ./test.sh
 
+      - name: Run Bindgen Test
+        run: |
+          curl -L https://github.com/dylibso/xtp-bindgen-test/releases/download/0.0.1/bundle.zip > bundle.zip
+          unzip bundle.zip
+          cd bundle/
+          xtp plugin init --schema-file schema.yaml --language TypeScript --path exampleplugin --feature stub-with-code-samples
```

Comment:
This should point at the current bundle, not the live deployed plugin which is what  `--language` does